packages/@okta/vuepress-site/docs/guides/custom-url-domain/main/index.md

```diff
@@ -61,7 +57,51 @@ No. You can only set up one custom domain per Okta org.
 
 Yes. When you turn the custom domain on, the Okta domain (for example, `example.okta.com`) still works.
 
-## Validate your TLS certificate
+<ApiLifecycle access="ea" />
```

Comment:
The way that this tag renders on the page makes it look like it applies to the Q&A just before the custom domain section. I wonder if we could tweak the tag's placement somehow so that it's clear that it applies to custom domain?